charts/adguard-home/README.md

```diff
@@ -0,0 +1,73 @@
+# DNS proxy as ad-blocker for local network
+
+This is an opinionated helm chart for [adguard-home](https://github.com/AdguardTeam/AdGuardHome)
+
+The default values and container images used in this chart will allow for running in a multi-arch cluster (amd64, arm, arm64)
+
+## TL;DR
+
+```shell
+helm repo add billimek https://billimek.com/billimek-charts/
+helm install billimek/adguard-home
+```
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```console
+helm install --name adguard-home billimek/adguard-home
+```
+
+## Uninstalling the Chart
+
+To uninstall/delete the `adguard-home` deployment:
+
+```console
+helm delete adguard-home --purge
+```
+
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## Configuration
+
+Read through the [values.yaml](https://github.com/billimek/billimek-charts/blob/master/charts/adguard-home/values.yaml) file. It has several commented out suggested values.
+
+Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example,
+
+```console
+helm install --name adguard-home \
+  --set timeZone="America/New York" \
+    billimek/adguard-home
+```
+
+Alternatively, a YAML file that specifies the values for the above parameters can be provided while installing the chart. For example,
+
+```console
+helm install --name adguard-home -f values.yaml billimek/adguard-home
+```
+
+## Upgrading an existing Release to a new major version
+
+A major chart version change (like 2.2.2 -> 3.0.0) indicates that there is an
+incompatible breaking change needing manual actions.
+
+### Upgrading from 2.x.x to 3.x.x
+
+Due to the renaming of the service port, an upgrade-in-place will not work.  The following are possible approaches to solve this:
```

Comment:
Let's remove this section until/unless we have to deal with a breaking change and specific instructions for that.

Reply:
Fixed them all in the newest commit.